docs/getting-started/user-roadmap-zowe-explorer.md

```diff
@@ -1,121 +1,59 @@
-
-* [**Conformance Program**](../extend/zowe-conformance-program.md)
-
-  This topic introduces the Zowe Conformance Program. Conformance provides Independent Software Vendors (ISVs), System Integrators (SIs), and end users greater confidence that their software will behave as expected. As vendors, you are invited to submit conformance testing results for review and approval by the Open Mainframe Project. If your company provides software based on Zowe CLI, you are encouraged to get certified today.
-
-* [**Blog: Zowe Conformance Program Explained**](https://medium.com/zowe/zowe-conformance-program-7f1574ade8ea)
-
-   This Medium article provide more details about the Conformance Program, including useful references.
-
-## Troubleshooting and support
-
-* [**Troubleshooting Zowe Explorer**](../troubleshoot/ze/troubleshoot-ze.md)
-
-   Learn about the tools and techniques that are available to help you troubleshoot and resolve problems. You can also find the list of Zowe Explorer issues.
+- Review the [**Contribution Guidelines**](https://github.com/zowe/vscode-extension-for-zowe/blob/master/CONTRIBUTING.md) for a summary of conventions and best practices for development of the Visual Studio Code Extension for Zowe.
 
-* [**Submit an issue**](https://github.com/zowe/zowe-explorer-vscode/issues)
+## Zowe Explorer community resources
 
-   If you have an issue that is specific to Zowe Explorer, you can submit an issue against the `vscode-extension-for-zowe` repository.
+The Zowe ecosystem is more than a collection of applications and extensions. An entire community exists to work on enhancements, help answer questions, and discuss plans for the future of the open source project.
 
-## Community resources
+- Join the [**#zowe-explorer-vsce**](https://app.slack.com/client/T1BAJVCTY/CUVE37Z5F) Slack channel to ask questions, propose new ideas, and interact with the Zowe community.
 
-* [**Slack channel**](https://openmainframeproject.slack.com/)
+- You can join **Zowe Explorer squad meetings** to get involved. The meeting schedule is posted in the [Zowe calendar](https://lists.openmainframeproject.org/g/zowe-dev/calendar).
 
-   Join the `# zowe-explorer` Slack channel to ask questions, propose new ideas, and interact with the Zowe community.
+- Read informative [**blog posts about Zowe**](https://medium.com/zowe) on Medium to explore use cases, best practices, and more.
 
-* [**Zowe Explorer squad meetings**](https://lists.openmainframeproject.org/g/zowe-dev/calendar)
+- **Discuss Zowe topics** in the [Open Mainframe Project community forums](https://community.openmainframeproject.org/c/zowe).
 
-   You can join one of the Zowe Explorer squad meetings to get involved.
+- If you have an issue that is specific to Zowe Explorer, you can [**submit an issue**](https://github.com/zowe/zowe-explorer-vscode/issues/new/choose) in the `vscode-extension-for-zowe` repository on GitHub.
```

Comment:
The GitHub repo has been renamed to `zowe-explorer-vscode`. The link for submitting an issue is still valid.